controller/apps_authclient.go

```diff
@@ -0,0 +1,172 @@
+package controller
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	goajwt "github.com/goadesign/goa/middleware/security/jwt"
+	rest "k8s.io/client-go/rest"
+)
+
+// AuthClient contains configuration and methods for interacting with OSIO API
+type AuthClient struct {
```

Comment:
I'm not sure you need all this here, since there's already an `auth/authservice` package with a client for the `auth` service.

Reply:
I looked into this - it's got a very old version of Auth checked in.  For example, it's missing the Cluster field on the UserData, and it's missing the /api/token?for= API call.
I propose removing controller/api_authclient in a later patch, after auth/authclient has been updated.

Reply:
The correct fix is to update the Auth in the Glide files, but I haven't looked into this.

Reply:
Yes, you need to update auth. We didn't break any old API in auth. So, updating Auth is pretty safe.

Reply:
this is now done.